### PR TITLE
Notes update

### PR DESCRIPTION
**Pull request by elfametesar**
Clearing all notes added, shows no notes now if you have none, others can call your notes via #

### CODE REVIEW

File: notes.py [hunk 1]
```diff
@@ -39,7 +39,8 @@ class NotesMod(loader.Module):
                "notes_header": "<b>Saved notes:</b>\n\n",
                "notes_item": "<b>•</b> <code>{}</code>",
                "delnote_args": "<b>What note should be deleted?</b>",
-               "delnote_done": "<b>Note deleted</b>"}
+               "delnote_done": "<b>Note deleted</b>", "delnotes_none": "<b>There are no notes to be cleared</b>",
```
Comment (penn5) [inline notes.py]: One per line please

File: notes.py [hunk 2]
```diff
@@ -57,6 +58,14 @@ def config_complete(self):
             return
         await utils.answer(message, await self._db.fetch_asset(asset_id))
 
+    async def delnotescmd(self, message):
```
Comment (penn5) [inline notes.py]: Better call it delallnotes

File: notes.py [hunk 3]
```diff
@@ -90,9 +99,21 @@ def config_complete(self):
...
+            await utils.answer(message, self.strings["notes_none"])
+            return
         await utils.answer(message, self.strings["notes_header"]
                            + "\n".join(self.strings["notes_item"].format(key)
                            for key in self._db.get(__name__, "notes", {})))
 
+    async def watcher(self, message):
+        args = message.text
+        notes = self._db.get(__name__, "notes", {})
+        if args.startswith("#"):
```
Comment (penn5) [inline notes.py:114]: Jmm

**Review by penn5 — CHANGES_REQUESTED** [review]

File: notes.py (hunk 1 above)
Comment (elfametesar) [reply notes.py]: lmao i should've seen this coming

Comment (elfametesar) [reply notes.py]: not sure if i could edit in on github tho, might break lint

**Review by penn5 — APPROVED** [review]

**Pull request merged**